src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolJrdOfficeOrgRoleMappingTest.java

```diff
@@ -159,6 +161,46 @@ void shouldReturnTribunalJudgeSalariedRoles_withIAC() {
         });
     }
 
+    @Test
+    @DisplayName("Scenario 4: IAC Judge with Non IAC Auth")
+    void iacJudgeWithoutIacAuth() {
+
+        judicialAccessProfiles.forEach(judicialAccessProfile -> {
+            judicialAccessProfile.setAppointment("Tribunal Judge");
+            judicialAccessProfile.setAppointmentType("Salaried");
+            judicialAccessProfile.setServiceCode("BFA1");
+            judicialAccessProfile.setTicketCodes(List.of("35970"));
```

Comment:
Can you please change this to 373. BFA1's ticket code is 373 and all ticket codes will be 3 digits

Reply:
ok will do

Reply:
this was in the excel sheet scenarios to be honest

but i will change it to suit